src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1

```diff
@@ -78,7 +78,7 @@ function Test-VirtualNetworkGatewayConnectionCRUD
       Assert-AreEqual 1 @($list).Count
       Assert-AreEqual $list[0].ResourceGroupName $actual.ResourceGroupName
       Assert-AreEqual $list[0].Name $actual.Name	
-      Assert-AreEqual $list[0].Location $actual.Location
+      # Assert-AreEqual $list[0].Location $actual.Location
```

Comment:
I had to re-record N/w tests and spoke to the dev owner for fixing these tests - all changes to n/w tests are as per his recommendation